PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,111 @@
+# Description
+> Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context and any links to documentation created.
+> Reference: [PR Submission Guide](/submission_process.md)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
```

Comment:
Two sections I think shoud be somewhere at the top (not necessarily here at the very top though, up to you)

 - Have any storage slots changes as a result of this PR?
 - Have any interfaces changed as a result of this PR?